### ISSUE TITLE
Object completion

### ISSUE THREAD

**Issue by wilzbach** [storyscript/sls#136]
e.g. `app.` or objects from services

**Comment by stevepeak** [comment]
`app.` may be deprecated, TBD on how we generate variables in the Studio. Let's only focus on objects from services here.

**Comment by wilzbach** [comment]
Good to know, but it's the same logic as all of these symbols are in the local scope. In fact, SLS is completely agnostic to `app`, it only cares about the symbol table provided by Storyscript.

So what the object completion would do is very similar to the existing mutation completion:
- identify current scope block (SLS) -> parse it and return the symbol table (this is done by Storyscript)
- identify name of object
- lookup word in symbol table
- STOP if no hit
...

tl;dr: at the moment Storyscript (compiler) adds `app` to the symbol table. If we decide to remove it, SLS would propagate this for its completion.
